e2e/awslogs_test.go

```diff
@@ -62,28 +258,46 @@ var testAwslogs = func() {
 				awslogsGroupKey:               testAwslogsGroup,
 				awslogsStreamKey:              testAwslogsStream,
 				awslogsEndpointKey:            testAwslogsEndpoint,
+				awslogsCreateGroupKey:         "false",
+				awslogsCreateStreamKey:        "false",
 			}
 			creator := cio.BinaryIO(*Binary, args)
-			sendTestLogByContainerd(creator, testLog)
-			validateTestLogsInAwslogs(cwClient, testAwslogsGroup, testAwslogsStream, testLog)
+			err := sendTestLogByContainerd(creator, testLog)
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
+			err = validateTestLogsInAwslogs(cwClient, testAwslogsGroup, testAwslogsStream, []string{testLog})
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
 		})
 	})
 }
 
-func validateTestLogsInAwslogs(client *cloudwatchlogs.Client, logGroupName string, logStreamName string, testLog string) {
+func validateTestLogsInAwslogs(client *cloudwatchlogs.Client, logGroupName string, logStreamName string, testLogs []string) error {
```

Comment:
nit: nonblocking but style preference, call site will look cleaner
```suggestion
func validateTestLogsInAwslogs(client *cloudwatchlogs.Client, logGroupName string, logStreamName string, testLogs ...string) error {
```

Reply:
I also want to the do the slice length matching so slice may be more intuitive.

Reply:
ok, FWIW, you can still do length matching:

```go
func main() {
	s := []string{"runfinch", "hello"}
	s1 := "finch"
	s2 := "world"

	strslice(s, s1, s2)
}

func strslice(slc []string, s ...string) {
	fmt.Println(len(slc))
	fmt.Println(len(s))
	fmt.Println(slc)
	fmt.Println(s)
}
```

```
2
2
[runfinch hello]
[finch world]
```

Reply:
Yeah. I just feel the meaning of slice aligns more, which specifies a set arg of similar elements, but in a lower layer than other args in the function. But call site is more like multiple elements in the same layer with other args in the function and we just don't specify the exact number there, which is usually for `..args`.

The set of all the test logs seems in the same layer as other args, like client, group name, stream name. But each test log does not.

Reply:
> Yeah. I just feel the meaning of slice aligns more, which specifies a set arg of similar elements

Totally agree :)